bio_hansel/main.py

```diff
@@ -246,8 +251,11 @@ def main():
 
     if output_tile_results:
         if len(dfs) > 0:
-            dfall = pd.concat(dfs)  # type: pd.DataFrame
-            dfall = dfall.sort_values(by='is_pos_tile', ascending=False)
+            dfall = pd.concat(dfs, sort=True)  # type: pd.DataFrame
+            if directory_path == None:
+                dfall = dfall.sort_values(by='is_pos_tile', ascending=False)
```

Comment:
Are you checking if multiple input have been specified and if so, don't perform sorting? What about presorting each DF in `dfs` prior to `pd.concat`?

Reply:
I was having trouble getting the sort done before concatenating the datasets so I decided to push it as is and work more at it. I'm currently seeing what I can do to do it this way

Reply:
How about?

```python
dfall = pd.concat([df.sort_values('is_pos_tile', ascending=False) for df in dfs], sort=False)
```

Reply:
Yes that does indeed work. Thanks, changes made